mne/source_space.py

```diff
@@ -1557,6 +1571,14 @@ def setup_volume_source_space(subject=None, pos=5.0, mri=None,
     if bem is not None and surface is not None:
         raise ValueError('Only one of "bem" and "surface" should be '
                          'specified')
+
+    if (mri is None and subject is not None and
+            volume_label is None and isinstance(pos, (float, int))):
+        mri = 'T1.mgz'
+
+    if volume_label is not None and mri == 'T1.mgz':
```

Comment:
I have noticed that this if-statement is similar to the one on 1594 (1572), do you think they can be combined?